app/tests/hanging_protocols_tests/test_forms.py

```diff
@@ -666,3 +669,78 @@ def test_generate_view_content_example(
     )
 
     assert view_content_example_json == expected_example_json
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize(
+    "hanging_protocol_json,view_content_example_json_expected_keys",
```

Comment:
The test function creates a civ_list with 4 component interfaces that can be set in a view port (images+isolated). The parameterized test covers the following cases:

- No hanging protocol is provided (None)
- No viewports in the hanging protocol json
- The hanging protocol viewports are less than the number of CIs (images+isolated)
- The hanging protocol viewports are equal to the number of CIs (images+isolated)

Regarding this case:
- The hanging protocol viewports are larger than the number of CIs (images+isolated)
This will never pass since the validation condition is that the keys of the view_content example json are the same as the viewports of the hanging protocol json.
When the hanging protocol viewports are larger than the number of CIs, the generated example will not have the same keys and will always raise RuntimeError.
What do you suggest for this case? we can't add more CIs to the example if none are there, so I am not sure about covering this case.

Also, regarding the case:
- No viewports in the hanging protocol json
I found a bug where the user can provide the following value for the hanging protocol json "[{}]" and it will pass the validation but it does not have the key "viewport_name" in the json and that will lead to an exception.
This issue is not related to the content_view example, and I will approach it later once I am done with my pitch.

Reply:
> * When the hanging protocol viewports are larger than the number of CIs, the generated example will not have the same keys and will always raise RuntimeError.

Hm, I guess the only option in that case is to return None as example then, right? @miriam-groeneveld, could you comment?

Reply:
Or is it possible to add an empty viewport to the `view_content`?

Reply:
What about re-using the CI's? I think that's more helpful. For example image A with overlay X on main, image A with overlay Y on secondary would be an example for this use-case.

Reply:
Ah of course, that makes a lot more sense! Thanks, Miriam!